README.md

```diff
@@ -46,168 +49,171 @@ With their functions being:
+`est`           |quarter and year analysts earnings estimates ([Business Insider](https://www.businessinsider.com/))
+`ins`           |insider activity over time ([Business Insider](https://www.businessinsider.com/))
+`insider`       |insider trading of the company ([Finviz](https://finviz.com/))
+`sec`           |SEC filings ([MarketWatch](https://www.marketwatch.com/))
+`short`         |short interest ([Quandl](https://www.quandl.com/))
+`warnings`      |company warnings according to Sean Seah book ([MarketWatch](https://www.marketwatch.com/))
+
 
 ### [Prediction Techniques](prediction_techniques/README.md) <a name="Prediction_Techniques"></a>
-```
-sma         simple moving average
-knn         k-Nearest Neighbors
-linear      linear regression (polynomial 1)
-quadratic   quadratic regression (polynomial 2)
-cubic       cubic regression (polynomial 3)
-regression  regression (other polynomial)
-arima       autoregressive integrated moving average
-prophet     Facebook's prophet prediction
-mlp         MultiLayer Perceptron
-rnn         Recurrent Neural Network
-lstm        Long-Short Term Memory
-```
+Command|Algorithm
+------ | ----------
+`sma`         |simple moving average ([source 1](https://en.wikipedia.org/wiki/Moving_average#Simple_moving_average), [source 2](https://www.investopedia.com/terms/s/sma.asp))
+`knn`         |k-Nearest Neighbors ([source 1](https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm))
+`linear`      |linear regression (polynomial 1) ([source 1](https://en.wikipedia.org/wiki/Linear_regression), [source 2](https://www.investopedia.com/terms/r/regression.asp))
+`quadratic`   |quadratic regression (polynomial 2) ([source 1](https://en.wikipedia.org/wiki/Polynomial_regression), [source 2](https://www.investopedia.com/terms/r/regression.asp))
+`cubic`       |cubic regression (polynomial 3) ([source 1](https://en.wikipedia.org/wiki/Polynomial_regression), [source 2](https://www.investopedia.com/terms/r/regression.asp))
+`regression`  |regression (other polynomial) ([source 1](https://en.wikipedia.org/wiki/Polynomial_regression), [source 2](https://www.investopedia.com/terms/r/regression.asp))
+`arima`       |autoregressive integrated moving average ([source 1](https://en.wikipedia.org/wiki/Autoregressive_integrated_moving_average), [source 2](https://www.investopedia.com/terms/a/autoregressive-integrated-moving-average-arima.asp)) 
+`prophet`     |Facebook's prophet prediction] ([source 1](https://facebook.github.io/prophet/))
+`mlp`         |MultiLayer Perceptron ([source 1](https://en.wikipedia.org/wiki/Multilayer_perceptron))
+`rnn`         |Recurrent Neural Network  ([source 1](https://en.wikipedia.org/wiki/Recurrent_neural_network))
+`lstm`        |Long Short-Term Memory  ([source 1](https://en.wikipedia.org/wiki/Long_short-term_memory))
+
 
 ## Install <a name="Install"></a>
 
 This project was written and tested with Python 3.6.8.
```

Comment:
I don't think this is accurate anymore. `requirements.txt` specifies `sciPy=1.6.1` which requires `python > 3.7`

Reply:
Shall I add that to this PR?

Reply:
Thought it makes sense as you're already update/improve the readme.

Reply:
Can't update this PR anymore since I accidentally deleted the fork repo (meant to delete a fork of another repo from DidierRLopes). I'll wait for this to get merged/not merged before making another to avoid merge conflicts..

Reply:
Hey @nechris , well spotted. This is because we don't have smoke-tests yet in place and I introduced a regression when reviewing a PR.

The code has indeed been tested with 3.6.8, but the scipy version was 1.4.1. Sorry for that! This will be fixed really soon :)

In the future we intend on not having such silly mistakes. Bear with us for the moment!